ads/inabox/inabox-messaging-host.js

```diff
@@ -200,11 +198,10 @@ export class InaboxMessagingHost {
    * @param {string} origin
```

Comment:
Remove this line.

Reply:
Also linter is complaining about some reformatting: https://travis-ci.org/ampproject/amphtml/jobs/608264619?utm_medium=notification&utm_source=github_status